### PR TITLE
Update 06-reproj.Rmd

### PR DESCRIPTION
**Pull request by katygregg**
:)

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Thanks!

File: 06-reproj.Rmd [hunk 1]
```diff
@@ -54,12 +54,12 @@ london_buff = st_buffer(london_geo, dist = 1)
...
 
 Only the second operation generates a warning.
-The warning message it useful, telling us that result may be of limited use because it is in units of latitude and longitude, rather than meters or some other suitable measure of distance assumed by `st_buffer()`.
+The warning message is useful, telling us that the result may be of limited use because it is in units of latitude and longitude, rather than meters or some other suitable measure of distance assumed by `st_buffer()`.
 The consequences of a failure to work on projected data are illustrated in Figure \@ref(fig:crs-buf) (left panel):
 the buffer is elongated in the north-south direction because lines of longitude converge towards the Earth's poles.
 
 ```{block2 type="rmdnote"}
-The distance between two lines of longitude, called meridians, is around 111 km at the equator (execute `geosphere::distGeo(c(0, 0), c(1, 0))` to find the precise distance).
+The distance between two lines of longitude, called meridians, is around 111 at the equator (execute `geosphere::distGeo(c(0, 0), c(1, 0))` to find the precise distance).
```
Comment (Nowosad) [inline 06-reproj.Rmd]: @katygregg @Robinlovelace I think it should be "111 km".

Comment (Robinlovelace) [reply 06-reproj.Rmd]: Well spotted. Fixed in https://github.com/Robinlovelace/geocompr/commit/07af0250820693b14954a698048a8c338f543202

Comment (katygregg) [reply 06-reproj.Rmd]: Oops sorry, that was a mistake :)

Comment (Nowosad) [reply 06-reproj.Rmd]: No problemo.
Thank you @katygregg for your great work!

**Pull request merged**